packages/apollos-data-connector-rock/src/content-items/__tests__/data-source.tests.js

```diff
@@ -693,6 +693,9 @@ describe('ContentItemsModel', () => {
     dataSource.getCursorByChildContentItemId = () => ({
       get: () => Promise.resolve([{ attributeValues: {}, attributes: {} }]),
     });
+    dataSource.request = () => ({
```

Comment:
maybe something like this to fix the test?

```suggestion
    dataSource.request = (path) => path.include('ContentChannels') ? ({ get:() => null }) : ({
```

Reply:
niiiice, yea that's smart